packages/platforms/browser/lib/config.ts

```diff
@@ -79,6 +80,11 @@ export function createSchema (hostname: string, defaultRoutingProvider: RoutingP
       defaultValue: defaultSendPageAttributes,
       message: 'should be an object',
       validate: isSendPageAttributes
+    },
+    tracePropagationUrls: {
+      defaultValue: [],
```

Comment:
Initially a default value of `[]` will be used so that it is disabled by default and we can publish this on the mainline - and enable in manually for dashboard and others as necessary. When we come to release proper, this will have a new defaultValue

> The default value of tracePropagationTargets is ['localhost', /^\//]. This means that by default, tracing headers are only attached to requests that contain localhost in their URL or requests whose URL starts with a '/' (for example GET /api/v1/users)